index.html

```diff
@@ -53,6 +53,17 @@ <h1 style="text-align: center; ">
     </div>
     <div class="row">
       <div class="col-md-offset-2 col-md-8">
+        <h4>Content</h4>
+        <ul>
+          <li><a href="#purpose">Purpose</a></li>
+          <li><a href="#management">Management</a></li>
+          <li><a href="#system-operations">System Operations</a></li>
+          <li><a href="#new"><strong>Creating a new identifier</strong></a></li>
```

Comment:
Capitalize like the rest.
```suggestion
          <li><a href="#new"><strong>Creating a New Identifier</strong></a></li>
```